yt/yt/server/http_proxy/bootstrap.cpp

```diff
@@ -267,7 +267,7 @@ TBootstrap::TBootstrap(TProxyConfigPtr config, INodePtr configNode)
 
     if (Config_->HttpsServer) {
         Config_->HttpsServer->ServerName = "HttpsApi";
-        ApiHttpsServer_ = NHttps::CreateServer(Config_->HttpsServer, Poller_, Acceptor_);
+        ApiHttpsServer_ = NHttps::CreateServer(Config_->HttpsServer, Poller_, Acceptor_, GetControlInvoker());
```

Comment:
Typically control invoker in YT process is used for some lightweight actions that should be properly serialized with this invoker (for example, in node, complex communication with master is performed in this thread). Sometimes this actions are latency-critical, so we expect this thread not to be loaded and all actions be quite fast.

Assuming this I'm not sure that control invoker is the best choice for a thread to work with filesystem, since disk may be overloaded. Maybe something like RpcHeavy suites better for that?

Reply:
HTTP-proxy executes disk-change-checker (whatever it is) in the same control-invoker.

If you want to offload all disk I/O into isolated thread then rpc-heavy seems wrong choice too.

Anyway most times certificate update ends with calling stat for couple files which metadata almost certainly will be in in cache or even in tmpfs.

Reply:
> HTTP-proxy executes disk-change-checker (whatever it is) in the same control-invoker.

Okay, in this case it's perferctly fine to leave control invoker.